Office 365/o365/ingest/parser.yml

```diff
@@ -42,7 +42,8 @@ stages:
           event.action: '{{json_event.message.Operation}}'
           event.kind: 'event'
           event.code: '{{json_event.message.RecordType | string}}'
-          user.name: '{{json_event.message.UserId}}'
+          event.reason: '{{json_event.message.ActionName}}'
+          user.name: '{{json_event.message.UserId or json_event.message.UserKey}}'
```

Comment:
`UserKey` seems to be an identifier (cf [doc](https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#common-schema)). Should not be moved in `user.id`?

Reply:
done